package.json

```diff
@@ -38,15 +34,15 @@
     ]
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^5.5.0",
+    "bootstrap": "^4.4.1",
```

Comment:
Should this be here?

Reply:
No, it should be under dependencies. I made the change.

Reply:
Only for the app's layout if it is on mobile, but I can take it out completely and just use react-bootstrap